### ISSUE TITLE
Search bar shows `Google Custom Search` placeholder

### ISSUE THREAD

**Issue by kushthedude** [istio/istio.io#8462]
**Description** 
The search bar in header nav-bar shows `Google Custom Search` occasionally when opened.

**Expected Behaviour**
`Custom Search` placeholder should not appear.

**Screenshot**
<img width="1323" alt="Screenshot 2020-11-09 at 5 36 59 AM" src="https://user-images.githubusercontent.com/44091822/98488164-9bfff080-224d-11eb-9980-65a0e0e1e4ce.png">

**Comment by frankbu** [comment]
Yes, I notice it too. It just seems to appear there sometimes, not sure what triggers it.

@davidhauck Can you look into this?

**Comment by davidhauck** [comment]
For me, it happens when I click on the search button and then click somewhere else on the page (not using the "x" button to close to search). I'll take a look when I get a chance

**Comment by kushthedude** [comment]
I can fix it, it happens when you hover over the cross button without clicking it.
